src/main/java/seedu/guest/ui/GuestCard.java

```diff
@@ -54,14 +59,112 @@ public GuestCard(Guest guest, int displayedIndex) {
         this.guest = guest;
         id.setText(displayedIndex + ". ");
         name.setText(guest.getName().fullName);
+        name.setWrapText(true);
+        Tooltip nameTT = new Tooltip("Name");
+        name.setTooltip(nameTT);
+
         phone.setText(guest.getPhone().value);
+        Image phoneImage = new Image(getClass().getResourceAsStream("/images/phone.png"));
+        ImageView setPhoneImage = new ImageView(phoneImage);
+        setPhoneImage.setFitWidth(16);
```

Comment:
Should you use a variable to store the value of 16 in setFitWidth instead of hard coding the number? As the value 16 is repeated multiple times, it might be easier to abstract it out so that it will be easier to edit in the future.

Reply:
Agreed and changed.